src/Extensions/Logging/InitialisationLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the proper ILogger from DI is set
+	/// not to be used as main logger
+	/// </summary>
+	public static class InitialisationLogger
+	{
+		/// <summary>
+		/// Instance of logger
+		/// </summary>
+		public static ILogger Instance { get; private set; } = LoggerFactory.Create(builder =>
+		 {
+			 builder.LoadInitialisationLogger();
+		 }).CreateLogger("Initial-Logging");
+
+		/// <summary>
+		/// Log on success
+		/// </summary>
+		/// <param name="serviceNameForLogging">Service name for logging</param>
+		/// <param name="message">Message to log</param>
+		public static void InitilisationSucceed(string serviceNameForLogging, string message = "")
```

Comment:
Sorry if I'm wrong here I'm not great with English, but should not it be `Initialization`?

Reply:
Oh yeah went with english spelling, thats the american can switch it to american version